roles/matrix-dendrite/defaults/main.yml

```diff
@@ -83,7 +83,7 @@ matrix_dendrite_rate_limiting_threshold: 5
 matrix_dendrite_rate_limiting_cooloff_ms: 500
 
 # Controls whether people with access to the homeserver can register by themselves.
-matrix_dendrite_registration_disabled: false
+matrix_dendrite_registration_disabled: true
```

Comment:
Huh! I wonder why this wasn't set to `true` to begin with. Looks like a huge omission